apps/examples/helloxx/Kconfig

```diff
@@ -6,7 +6,7 @@
 config EXAMPLES_HELLOXX
```

Comment:
In the PR description, did you mean "since c++ support was **NOT** available in previous docker versions"?